src/steps/updateAllContributorsTable.ts

```diff
@@ -20,5 +20,6 @@ export async function updateAllContributorsTable({
 		}),
 	);
 
+	await $`pnpm add all-contributors-cli -D`;
```

Comment:
This is uninstalled later in `finalizeDependencies`.

Reply:
I don't love the extra time spent installing this, but I'm going to be switching this whole situation over to the `create` engine soon anyway.

Reply:
Ah, migration failures...